assets/src/scripts/components/hydrograph/points.js

```diff
@@ -0,0 +1,57 @@
+const { createSelector } = require('reselect');
+
+const { xScaleSelector, yScaleSelector } = require('./scales');
+
+/**
+ * Returns the points for a given timeseries.
+ * @param  {Object} state     Redux store
+ * @param  {String} tsDataKey Timeseries key
+ * @return {Array}            Array of points.
+ */
+const pointsSelector = function (state, tsDataKey) {
+    if (state.tsData[tsDataKey]) {
+        return state.tsData[tsDataKey];
+    }
+    return [];
+};
+
+
+/**
+ * Returns the current show state of a timeseries.
+ * @param  {Object}  state     Redux store
+ * @param  {String}  tsDataKey Timeseries key
+ * @return {Boolean}           Show state of the timeseries
+ */
+const isVisibleSelector = function (state, tsDataKey) {
```

Comment:
Is there a reason anonymous functions are being used rather than named ones?

Reply:
Only so they can be marked as `const`. I don't mind switching to named functions if you or Mary dislike it.

Reply:
I'm not sure what this comment applies too. Are you talking about line 46?

Reply:
Not really. I was more referring the this situation:
```
const myFunc = funct(x, y) {
     do some stuff;
}
```
versus
```
function myfunc(x, y) {
    do some stuff;
}

Reply:
So I really am fried. So the question is why isn't this written as function isVisibleSelector(state, tsDataKey)? I don't mind it being assigned to a const. In fact, that's the pattern I tried to use when doing standard JS except had to use var.

Reply:
@mbucknell  -- That is pretty much the question.

I don't really mind the pattern in and of itself. It was more question aimed a probing the reason for using this pattern here, while the other modules seem to use `function someName()`.

Reply:
I think it also reinforces that functions are first class objects.